coordinator/src/services/translation/request/task.ts

```diff
@@ -1,18 +1,24 @@
 import {fhir, hl7V3} from "@models"
-import {getIdentifierValueForSystem} from "../common"
+import {getExtensionForUrl, getIdentifierValueForSystem} from "../common"
 import * as pino from "pino"
-import {createAuthorForUnattendedAccess} from "./agent-unattended"
+import {createAuthorForAttendedAccess} from "./agent-attended"
 
-export async function createAuthorFromTaskOwnerIdentifier(
-  identifier: fhir.Identifier,
+export async function createAuthorFromProvenanceAgentExtension(
```

Comment:
name's a bit confusing here, it takes a task as input and uses two parts of it. maybe rename to `createAuthorFromTask`?